core-java-modules/core-java-lang-4/src/main/java/com/baeldung/bookstore/Book.java

```diff
@@ -0,0 +1,17 @@
+//incorrect package com.baeldung declaration;
+package com.baeldung.bookstore;
+
+public class Book {
+
+	private String title;
+	private String author;
+	private long isbn;
+
+	public Book(final String title, final String author, final long isbn) {
+		super();
```

Comment:
Unnecessary call to `super()` - please remove

Reply:
Have removed the super call.